elastalert/elastalert.py

```diff
@@ -456,8 +456,21 @@ def get_hits_count(self, rule, starttime, endtime, index):
             five=rule['five']
         )
 
+        es_client = self.thread_data.current_es
         try:
-            res = self.thread_data.current_es.count(index=index, doc_type=rule['doc_type'], body=query, ignore_unavailable=True)
+            if es_client.is_atleastsixtwo():
```

Comment:
This check is done in a handleful of other places:

https://search.d2l.dev/search?project=Brightspace&full=&defs=&refs=is_atleastsixtwo&path=elastalert2%2Felastalert.py&hist=&type=&xrd=&nn=1&si=refs&si=refs